addons/sourcemod/scripting/shop/item_manager.sp

```diff
@@ -1371,19 +1371,15 @@ public int ItemManager_FormatItemNative(Handle plugin, int numParams)
 	return true;
 }
 
-bool ItemManager_FillCategories(Menu menu, int source_client, bool inventory = false, bool showAll = false)
+bool ItemManager_FillCategories(Menu menu, int source_client, bool showAll = false, ShopMenu shop_menu)
```

Comment:
Sorry, I didn't see it right away. Parameters with default values must be last in the list of parameters

```
bool ItemManager_FillCategories(Menu menu, int source_client, ShopMenu shop_menu, bool showAll = false)
```